raiden_contracts/data_0.14.0/source/lib/SHA256.sol~

```diff
@@ -0,0 +1,3 @@
+pragma solidity 0.5.4;
```

Comment:
What is this for?

Reply:
That was my git mistake.  Added a whole directory without looking what's in.